dex/src/main/scala/com/wavesplatform/dex/actors/address/BatchOrderCancelActor.scala

```diff
@@ -9,24 +9,26 @@ import com.wavesplatform.dex.domain.order.Order
 import com.wavesplatform.dex.domain.utils.ScorexLogging
 import com.wavesplatform.dex.error
 
+import scala.collection.immutable.ListMap
 import scala.concurrent.duration.FiniteDuration
 
 class BatchOrderCancelActor private (
-  orderIds: Set[Order.Id],
+  initialResponse: ListMap[Order.Id, OrderCancelResult],
   source: Command.Source,
   processorActor: ActorRef,
   clientActor: ActorRef,
-  timeout: FiniteDuration,
-  initResponse: Map[Order.Id, OrderCancelResult]
+  timeout: FiniteDuration
 ) extends Actor
     with ScorexLogging {
 
   import BatchOrderCancelActor._
   import context.dispatcher
 
-  orderIds.foreach(processorActor ! CancelOrder(_, source))
+  initialResponse.foreach {
```

Comment:
I think, we could do better. This looks hacky

Reply:
fixed